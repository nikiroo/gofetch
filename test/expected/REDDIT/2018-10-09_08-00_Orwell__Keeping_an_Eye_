            ORWELL: KEEPING AN EYE ON YOU - FREE ON STEAM          

  [linux_gaming] 2018-10-09_08-00

  o Reference: 2018-10-09_08-00_Orwell__Keeping_an_Eye_
  o News link: https://www.reddit.com/r/linux_gaming/comments/9l3m6y/orwell_keeping_an_eye_on_you_free_on_steam/
  o Source link: https://store.steampowered.com/app/491950/Orwell_Keeping_an_Eye_On_You/


    free!  way  cheaper  than  a flight to london to experience the
    real thing.


  ** cal5582
     free! way cheaper than a flight to london to experience the real
     thing.

     ** ayyy_lmao2
        This was the most painful upvote I've given :(


  ** NoXPhasma
     [1]Be quick, offer valid only for 24 hours.
     
     
     
     
     [1]
     https://steamcommunity.com/games/491950/announcements/detail/337-
     0273013735458608

     ** Atlas__risen
        damn, I literally just bought it like four days ago.

        ** shaolinpunks
           If you've played it less than X amount of hours you can
           request a refund.

           ** Atlas__risen
              Nah, it's a good game. Let 'em keep the money.

              ** HighestExcitement
                 The right way to look at it. What's the full price?

                 ** breell
                    You can check HB, the standard price should be
                    the same:
                    [1]https://www.humblebundle.com/store/orwell
                    
                    
                    
                    
                    [1] https://www.humblebundle.com/store/orwell

                    ** HighestExcitement
                       Amazing they gave it for free. Open-minded
                       developers.

                       ** breell
                          They might be doing that to sell the new
                          extension.





              ** PM_ME_WRITING_ADVICE
                 respect.





     ** Garland_Key
        Done - looks like a great game too! THANK YOU!


  ** LionelSlothrop
     Thanks for the tip! I also want to mention i saw Hyperdrive
     Massacre is also free right now and is Linux compatible

     ** D4rCM4rC
        [1]Hyperdrive Massacre (Steam Store link for the lazy)
        
        
        
        
        [1] https://store.steampowered.com/app/402390/


     ** VictorVrine
        is there a sub where people post links for games that are
        temporarily free? what i used to do was subscribe to a bunch
        of shitty youtube channels who made videos about those but it
        never really works

        ** LionelSlothrop
           Yeah its called freegamefindings or something like that
           and gamedeals is another



  ** mylittlemoonfey
     This is a fun little game, I finished in one sitting.

  ** pr0ghead
     Wasn't that also free about a month ago? Or was that on GOG?

     ** NoXPhasma
        Yes, it was free on Humble Bundle. This is the chance to grab
        it for those who missed it.


  ** Offensive_joke_lord
     windows 10 simulator

  ** Reygle
     Shit, I paid for this a while back. It's not bad.

  ** NapoleonTheCake
     OP thank you very much!

  ** Espinosidro
     I want to know what does the FOSS community think about this
     game.

     ** ase1590
        I want to know what does the FOSS community think about this
        game.
        What about it? It's a game that has a message that advocates
        against government surveillance.

        ** Espinosidro
           Interesting interpretation, this is pretty much what I
           wanted to know.
           My question to you is, is it really advocating against
           surveillance? maybe it's trying to show us the security
           benefits of having no privacy.

           ** ase1590
              I've finished the game completely. You speak as if you
              have not.
              The whole message at the end was built to encourage
              you, the player, to seriously consider the downsides
              surveillance and the irresponsibility of those using
              the system can have on civilization. Though of course,
              you can totally ignore all of that rationale and simply
              keep blindly following orders from Catherine.

              ** Espinosidro
                 I have finished the game only once, haven't seen all
                 the endings yet. But as far as I can tell the game
                 isn't trying to simply tell the player that
                 surveillance is bad or anything, it just makes the
                 player think about it.
                 At the end of the day we can only guess what the
                 devs were thinking when making the game.

                 ** ase1590
                    But as far as I can tell the game isn't trying to
                    simply tell the player that surveillance is bad
                    or anything, it just makes the player think about
                    it.
                    Right. you're not being forced to pick a decision.
                    But you'd have a hard time telling me this game
                    was neutral or in favor of surveillance based on
                    several plot items in it. Especially with a name
                    like Orwell of all things.
                    You simply just don't name your game Orwell and
                    not make it lean anti-surveillance.






  ** MCHerb
     I was eventually going to buy this game, I'll wait till it's on
     sale :)

  ** drislands
     Oh fuck me I just bought this two days ago. Oh well, it was
     definitely worth the money. Excellent game, if a little short!

  ** CaptRobovski
     Reminds me of Papers, Please, in the sense that it's an admin
     simulator. Looks interesting!

  ** Draconicrose_
     I really liked this game. Definitely packs a but of an emotional
     punch.


