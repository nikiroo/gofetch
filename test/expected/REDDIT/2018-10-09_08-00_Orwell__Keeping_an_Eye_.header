0Orwell: Keeping an Eye On You - Free on Steam	null/REDDIT/2018-10-09_08-00_Orwell__Keeping_an_Eye_		70
i  [linux_gaming] 2018-10-09_08-00
i
