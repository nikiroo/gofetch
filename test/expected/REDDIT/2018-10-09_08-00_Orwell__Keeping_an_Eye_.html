<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Orwell: Keeping an Eye On You - Free on Steam</h2>
	<div class='details'>([linux_gaming] 2018-10-09_08-00)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>2018-10-09_08-00_Orwell__Keeping_an_Eye_</a></li>
        <li>News link: <a href='https://www.reddit.com/r/linux_gaming/comments/9l3m6y/orwell_keeping_an_eye_on_you_free_on_steam/'>https://www.reddit.com/r/linux_gaming/comments/9l3m6y/orwell_keeping_an_eye_on_you_free_on_steam/</a></li>
        <li>Source link: <a href='https://store.steampowered.com/app/491950/Orwell_Keeping_an_Eye_On_You/'>https://store.steampowered.com/app/491950/Orwell_Keeping_an_Eye_On_You/</a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		free! way cheaper than a flight to london to experience the real thing.
	</div>
<hr/>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>cal5582</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>free! way cheaper than a flight to london to experience the real thing.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>ayyy_lmao2</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>This was the most painful upvote I've given :(</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>NoXPhasma</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>[1]Be quick, offer valid only for 24 hours.</p><p></p><p></p><p></p><p></p><p>[1] https://steamcommunity.com/games/491950/announcements/detail/3370273013735458608</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Atlas__risen</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>damn, I literally just bought it like four days ago.</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h3>shaolinpunks</h3>
        <div class='by' style='font-style: italic;'></div>
        <div class='comment_content'><p>If you've played it less than X amount of hours you can request a refund.</p></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h3>Atlas__risen</h3>
          <div class='by' style='font-style: italic;'></div>
          <div class='comment_content'><p>Nah, it's a good game. Let 'em keep the money.</p></div>
          <div class='comment' style='display: block; margin-left: 80px'>
            <h3>HighestExcitement</h3>
            <div class='by' style='font-style: italic;'></div>
            <div class='comment_content'><p>The right way to look at it. What's the full price?</p></div>
            <div class='comment' style='display: block; margin-left: 80px'>
              <h3>breell</h3>
              <div class='by' style='font-style: italic;'></div>
              <div class='comment_content'><p>You can check HB, the standard price should be the same:</p><p>[1]https://www.humblebundle.com/store/orwell</p><p></p><p></p><p></p><p></p><p>[1] https://www.humblebundle.com/store/orwell</p></div>
              <div class='comment' style='display: block; margin-left: 80px'>
                <h3>HighestExcitement</h3>
                <div class='by' style='font-style: italic;'></div>
                <div class='comment_content'><p>Amazing they gave it for free. Open-minded developers.</p></div>
                <div class='comment' style='display: block; margin-left: 80px'>
                  <h3>breell</h3>
                  <div class='by' style='font-style: italic;'></div>
                  <div class='comment_content'><p>They might be doing that to sell the new extension.</p></div>
                </div>
              </div>
            </div>
          </div>
          <div class='comment' style='display: block; margin-left: 80px'>
            <h3>PM_ME_WRITING_ADVICE</h3>
            <div class='by' style='font-style: italic;'></div>
            <div class='comment_content'><p>respect.</p></div>
          </div>
        </div>
      </div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Garland_Key</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>Done - looks like a great game too! THANK YOU!</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>LionelSlothrop</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>Thanks for the tip! I also want to mention i saw Hyperdrive Massacre is also free right now and is Linux compatible</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>D4rCM4rC</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>[1]Hyperdrive Massacre (Steam Store link for the lazy)</p><p></p><p></p><p></p><p></p><p>[1] https://store.steampowered.com/app/402390/</p></div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>VictorVrine</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>is there a sub where people post links for games that are temporarily free? what i used to do was subscribe to a bunch of shitty youtube channels who made videos about those but it never really works</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h3>LionelSlothrop</h3>
        <div class='by' style='font-style: italic;'></div>
        <div class='comment_content'><p>Yeah its called freegamefindings or something like that and gamedeals is another</p></div>
      </div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>mylittlemoonfey</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>This is a fun little game, I finished in one sitting.</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>pr0ghead</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>Wasn't that also free about a month ago? Or was that on GOG?</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>NoXPhasma</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>Yes, it was free on Humble Bundle. This is the chance to grab it for those who missed it.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>Offensive_joke_lord</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>windows 10 simulator</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>Reygle</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>Shit, I paid for this a while back. It's not bad.</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>NapoleonTheCake</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>OP thank you very much!</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>Espinosidro</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>I want to know what does the FOSS community think about this game.</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>ase1590</h3>
      <div class='by' style='font-style: italic;'></div>
      <div class='comment_content'><p>I want to know what does the FOSS community think about this game.</p><p>What about it? It's a game that has a message that advocates against government surveillance.</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h3>Espinosidro</h3>
        <div class='by' style='font-style: italic;'></div>
        <div class='comment_content'><p>Interesting interpretation, this is pretty much what I wanted to know.</p><p>My question to you is, is it really advocating against surveillance? maybe it's trying to show us the security benefits of having no privacy.</p></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h3>ase1590</h3>
          <div class='by' style='font-style: italic;'></div>
          <div class='comment_content'><p>I've finished the game completely. You speak as if you have not.</p><p>The whole message at the end was built to encourage you, the player, to seriously consider the downsides surveillance and the irresponsibility of those using the system can have on civilization. Though of course, you can totally ignore all of that rationale and simply keep blindly following orders from Catherine.</p></div>
          <div class='comment' style='display: block; margin-left: 80px'>
            <h3>Espinosidro</h3>
            <div class='by' style='font-style: italic;'></div>
            <div class='comment_content'><p>I have finished the game only once, haven't seen all the endings yet. But as far as I can tell the game isn't trying to simply tell the player that surveillance is bad or anything, it just makes the player think about it.</p><p>At the end of the day we can only guess what the devs were thinking when making the game.</p></div>
            <div class='comment' style='display: block; margin-left: 80px'>
              <h3>ase1590</h3>
              <div class='by' style='font-style: italic;'></div>
              <div class='comment_content'><p>But as far as I can tell the game isn't trying to simply tell the player that surveillance is bad or anything, it just makes the player think about it.</p><p>Right. you're not being forced to pick a decision.</p><p>But you'd have a hard time telling me this game was neutral or in favor of surveillance based on several plot items in it. Especially with a name like Orwell of all things.</p><p>You simply just don't name your game Orwell and not make it lean anti-surveillance.</p></div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>MCHerb</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>I was eventually going to buy this game, I'll wait till it's on sale :)</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>drislands</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>Oh fuck me I just bought this two days ago. Oh well, it was definitely worth the money. Excellent game, if a little short!</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>CaptRobovski</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>Reminds me of Papers, Please, in the sense that it's an admin simulator. Looks interesting!</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>Draconicrose_</h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'><p>I really liked this game. Definitely packs a but of an emotional punch.</p></div>
  </div>
</div>
</body>
